.gitignore

```diff
@@ -0,0 +1,3 @@
+__pycache__
+venv
+tetris\ theme\ song.mp3
```

Comment:
This ".gitignore" file tells Github what files to track and which files to ignore. I added a few conventional ones.